docs/docs/running-node/observability.md

```diff
@@ -26,84 +26,5 @@ Traces are produced for all major pieces of work when processing a job, although
 ## Viewing
 The metrics and traces can easily be forwarded to a variety of different services as we use OpenTelemetry, such as Honeycomb or Datadog.
 
-To view the data locally, or simply to not use a SaaS offering, you can start up Jaeger and Prometheus placing these three files into a directory then running `docker compose start` while running Bacalhau with the
-`OTEL_EXPORTER_OTLP_PROTOCOL=grpc` and `OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317` environment variables.
-
-```yaml title="docker-compose.yaml"
-version: "2"
-services:
-
-  jaeger-all-in-one:
-    image: "jaegertracing/all-in-one:1.42"
-    restart: "always"
-    ports:
-      - "16686:16686" # Jaeger UI
-      - "14250:14250" # Jaeger gRPC endpoint
-
-  otel-collector:
-    image: "otel/opentelemetry-collector:0.70.0"
-    restart: "always"
-    command: ["--config=/etc/otel-collector-config.yaml"]
-    volumes:
-      - "./otel-collector-config.yaml:/etc/otel-collector-config.yaml"
-    ports:
-      - "8888:8888"   # Prometheus metrics exposed by the collector
-      - "8889:8889"   # Prometheus exporter metrics
-      - "13133:13133" # health_check extension
-      - "4317:4317"   # OTLP gRPC receiver
-    depends_on:
-      - "jaeger-all-in-one"
-      - "prometheus"
-
-  prometheus:
-    container_name: "prometheus"
-    image: "prom/prometheus:v2.42.0"
-    restart: "always"
-    volumes:
-      - "./prometheus.yaml:/etc/prometheus/prometheus.yml"
-    ports:
-      - "9090:9090" # Prometheus UI
-```
-
-```yaml title="otel-collector-config.yaml"
-receivers:
-  otlp:
-    protocols:
-      grpc:
-
-exporters:
-  prometheus:
-    endpoint: "0.0.0.0:8889"
-
-  jaeger:
-    endpoint: jaeger-all-in-one:14250
-    tls:
-      insecure: true
-
-processors:
-  batch:
-
-extensions:
-  health_check:
-
-service:
-  extensions: [health_check]
-  pipelines:
-    traces:
-      receivers: [otlp]
-      processors: [batch]
-      exporters: [jaeger]
-    metrics:
-      receivers: [otlp]
-      processors: [batch]
-      exporters: [prometheus]
-```
-
-```yaml title="prometheus.yaml"
-scrape_configs:
-  - job_name: 'otel-collector'
-    scrape_interval: 10s
-    static_configs:
-      - targets: ['otel-collector:8889']
-      - targets: ['otel-collector:8888']
-```
+To view the data locally, or simply to not use a SaaS offering, you can start follow the directions 
+[here](../../../ops/metrics/README.md)
```

Comment:
Does this need to be a github link rather than relative to docs?

Reply:
I can't link to github because this file doesn't exist there yet, I just added it in this PR.

Reply:
I can't link it to github becasue its a doc I added in this PR